tutor/models/course.py

```diff
@@ -0,0 +1,29 @@
+from .. import db
+
+
+class Course(db.Model):
+    __tablename__ = 'course'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(100), nullable=False)
+    resources = db.relationship('Resource', backref='course', lazy=True)
+
+    def __init__(self, name):
+        self.name = name
+        
```

Comment:
I don't think this file is passing flake8 validation

Reply:
@lioramilbaum I fixed the lint , and linked a matching issue